src/functions/searchResults/framework/database/query-builder.ts

```diff
@@ -22,9 +22,16 @@ export const getConciseSearchResultsFromSearchQuery = (queryParameters : QueryPa
     queries.push('staff_number = ?');
     parameterArray.push(queryParameters.staffNumber);
   }
+
   if (queryParameters.applicationReference) {
-    queries.push('application_reference = ?');
-    parameterArray.push(queryParameters.applicationReference.toString());
+    if (queryParameters.applicationReference.toString().length === 8) {
+      queries.push('application_reference >= ? AND application_reference <= ?');
+      parameterArray.push(`${queryParameters.applicationReference.toString()}000`);
```

Comment:
Can we add a comment why we are doing this and not using a Like or something else